core-bundle/src/Resources/contao/library/Contao/Model/QueryBuilder.php

```diff
@@ -45,9 +45,9 @@ public static function find(array $arrOptions)
 			foreach ($objBase->getRelations() as $strKey=>$arrConfig)
 			{
 				// Automatically join the single-relation records
-				if ($arrConfig['load'] == 'eager' || $arrOptions['eager'])
+				if (($arrConfig['load'] ?? null) == 'eager' || ($arrOptions['eager'] ?? false))
```

Comment:
```suggestion
				if (($arrConfig['load'] ?? null) == 'eager' || ($arrOptions['eager'] ?? null))
```

Using `?? false` is a behavior change again, even though I don't think it makes a difference. @ausi How would you handle this?

Reply:
I would use `?? null` for consistency but it is exactly the same in this case I think.

Reply:
I remember this discussion :upside_down_face:

Reply:
Changed in 9cd2ac02efc6ba2e8b36a22be0659c01e32c870d